R/auth.R

```diff
@@ -64,8 +48,21 @@ get_api_key <- function() {
 }
 
 #' @rdname get_api_key
-#' @param key API key to use for future requests
 #' @export
-set_api_key <- function(key) {
-  options(delphi.epidata.key = key)
+save_api_key <- function() {
+  cli::cli_inform(
+    c(
+      "i" = "This function will open your .Renviron file in a text editor. You will need to
+      write DELPHI_EPIDATA_KEY=yourkeyhere in the file and save it. If the editor
+      does not open, you will need to edit the file manually.",
+      "i" = "Press any key to continue."
+    ),
+  )
+  readline()
+
+  if (file.exists(usethis::proj_path(".Renviron"))) {
+    usethis::edit_r_environ(scope = "project")
```

Comment:
Should we print a message after each of these, saying which .Renviron was successfully edited? E.g. after project level, say "Editing project-level Renviron" and show the path to the project, so the user knows they only edited for that project

Reply:
`usethis::edit_r_environ` already does this when it returns!

Reply:
```
r$> usethis::edit_r_environ(scope="project")
✔ Setting active project to '/home/dshemeto/repos/covid-hosp-forecast'
• Modify '.Renviron'
• Restart R for changes to take effect

r$> usethis::edit_r_environ(scope="user")
• Modify '/home/dshemeto/.Renviron'
• Restart R for changes to take effect
```

Reply:
Very nice, great